### PR TITLE
Added Datek Contact Switch

### PR DESCRIPTION
**Pull request by cskogstrand**

### CODE REVIEW

File: devices/datek.js [hunk 1]
```diff
@@ -243,4 +243,19 @@ module.exports = [
         },
         exposes: [e.contact(), e.battery_low(), e.tamper(), e.temperature()],
     },
+    {
+        zigbeeModel: ['Contact Switch'],
+        model: 'Contact Switch',
```
Comment (Koenkk) [inline devices/datek.js:248]: Can you update this to the model listed on the site/device/box? Also update the picture in https://github.com/Koenkk/zigbee2mqtt.io/pull/2043/files after that.

Comment (cskogstrand) [reply devices/datek.js:248]: This is in fact the model name of the device :) Do you want me to change the name of the picture to DATEK_CONTACT_SWITCH maybe?

Screenshot 2023-05-20 at 18 42 15

Comment (cskogstrand) [reply devices/datek.js]: Just found the name you are after on the box :) HSE2936T
Changed it in commit [7dc7589](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5756/commits/7dc7589636e84baed22f19fde11a919a19b3c305)

Comment (cskogstrand) [reply devices/datek.js]: abbc9a7e9c984252e90d06aac74bca662930debced678eedb9544306bdce6ce1

**Comment by Koenkk** [comment]
Thanks, now it's ok, could you rebase this PR? (`datatek.js` was moved)

**Comment by cskogstrand** [comment]
Somehow this got closed in the process. Created a new PR with rebase.

**Pull request closed without merging**